packages/graphql/src/services/liveQuizzes.ts

```
@@ -198,6 +201,24 @@ async function getCachedBlockResults({
         total: parseInt(results.participants),
       } as ElementResultsOpen
     }
+    if (instance.elementType === ElementType.SELECTION) {
+      const selections = Object.entries(
+        omitBy(results, (_, key) => key === 'participants')
+      ).reduce<Record<string, number>>(
+        (acc, [answerId, count]) => {
+          return {
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)